src/components/cms/BotBuilder/BotBuilderWrap.js

```diff
@@ -1,27 +1,30 @@
 import React from 'react';
 import BotWizard from './BotWizard';
+import eventRegistration from '../../../../public/botTemplates/event-registration.jpg';
```

Comment:
Better to add `Image` prefix to all the images for better code readabiliity.

Reply:
okay